corehq/apps/es/transient_util.py

```diff
@@ -39,7 +44,16 @@ def doc_adapter_from_alias(index_alias, for_export=False):
     :param for_export: ``bool`` used to instantiate the adapter instance
     :returns: instance of an ``ElasticDocumentAdapter`` subclass
     """
-    return _DOC_ADAPTERS_BY_ALIAS[index_alias](for_export=for_export)
+    return _get_doc_adapter(_DOC_ADAPTERS_BY_ALIAS[index_alias], for_export)
+
+
+def _get_doc_adapter(adapter, for_export):
+    """Helper function to keep things DRY. Returns an adapter instance that is
+    configured for export (or not).
+    """
+    if for_export:
+        return adapter.export_adapter()
+    return adapter
```

Comment:
nit: seems like a good place for inline `if`
```suggestion
    return adapter.export_adapter() if for_export else adapter
```